industry/products.md

```diff
@@ -17,11 +17,13 @@ They vary by:
 and other features, as well as underlying architecture.
 
 
-An advantage of using a machine translation provider is that the service is already working and can be integrated immediately. An alternative to using a machine translation provider is to train a model from scratch or to use a pre-trained model, and deploy and maintain a machine translation system.
+Some of the most popular MT providers offer off-the-shelf solutions, i.e. trained MT models that are ready-to-use and can be integrated immediately. An alternative is to train a model from scratch, deploy it and maintain it.
 
-Unlike raw models, production machine translation systems usually provide language identification and sentence segmentation.
+The main advantage of ready-to-use systems is that they are a one-size-fits-all solution, for example they can be used for any text domain. When used in an application, text segmentation and language identification are usually provided. On the other hand, a model trained from scratch might yield better performance on a specific domain if trained on a large amount of high-quality domain-specific bilingual data. However, training and maintaining such models can be a complex task and it requires specific competences.
```

Comment:
I think we made a mistake to ever mention "advantages" - a bit subjective.

We should probably just list out the things that production systems can include, beyond what just the underlying model or research system includes, as a bullet-pointed list:

- language identification
- sentence segmentation
- bridging (pivoting)
- glossary
- encoding options
- HTML/XML tag handling

And most if not all of those should be linked to the relevant article.

And then we should also point out that:
- production systems usually support many languages
- it's on scalable infrastructure

Reply:
Added in the text